### ISSUE TITLE
Text in the profile property files is breaking profile imports.

### ISSUE THREAD

**Issue by Zithyxx** [runelite/runelite#16309]
### Type

Incorrect behavior

### Operating System

Windows

### OS Version

Windows 11

### Bug description

With the addition of the comments at the start of a profile properties text:

- #RuneLite configuration
- #[date & time the file was made]

It's completely broken importing profiles. When I import a profile to Runelite the client gets stuck on those two comment lines and won't do anything. If I delete them then save the profile properties file the import will work as it should.

### RuneLite version

Runelite version: 1.9.11.1
Launcher version: 2.6.2

**Comment by Felanbird** [comment]
Is it documented somewhere that adding comments to your `settings.properties` file is supported?

As far as I'm aware runelite contributors/helpers have been very vocal about not supporting manually editing said file, but it could be different for comments.

**Comment by Felanbird** [comment]
> I haven't added any comments to my profile properties files. The comments were automatically added when I exported a profile. I was trying to import the profile after I exported it, and noticed it wasn't working. When I inspected the file with Notepad++ I saw commends had been added in the first two lines of a profile properties.

I just noticed and was editing my comment, very odd 🤔

**Comment by Zithyxx** [comment]
I'm not sure when that functionality was added to the files, but it completely breaks the import process, the client gets stuck on those first two comments and won't import any settings from the file. Only after deleting them will the process work.

**Comment by Felanbird** [comment]
I finally got around to migrating to the profile system and unfortunately I had no issue importing my properties
The text in my properties was as follows:
```
#RuneLite configuration
#Tue Feb 21 20:30:43 EST 2023
```

Maybe some issue with the formatting? Is your version using a localized date? I'm not sure where the code is for said file but it would be my idea.
